test/performance/hack/crds/appliedmanifestwork.yaml

```diff
@@ -0,0 +1,123 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
can we use https://github.com/open-cluster-management-io/api/blob/bd6f2229b53cb61c76ebf528c73bc925d8e0f4c8/work/v1/0000_01_work.open-cluster-management.io_appliedmanifestworks.crd.yaml instead of include it in maestro?

Reply:
apply them from ocm directly